packages/tree-widget/package.json

```diff
@@ -0,0 +1,98 @@
+{
+  "name": "@bentley/tree-widget-react",
+  "version": "2.0.0",
+  "description": "Tree Widget React",
+  "license": "MIT",
+  "main": "lib/tree-widget-react.js",
+  "typings": "lib/tree-widget-react",
+  "scripts": {
+    "copy:assets": "cpx \"./src/**/*.scss\" ./lib && cpx \"./public/**/*\" ./lib/public",
+    "pseudolocalize": "betools pseudolocalize --englishDir ./public/locales/en --out ./lib/public/locales/en-PSEUDO",
+    "build": " tsc 1>&2 && npm run copy:assets && npm run pseudolocalize",
+    "build-prod": " tsc -p tsconfig.json 1>&2 && npm run copy:assets",
```

Comment:
This `build-prod` script won't be picked up automatically by this CI build. Do you think it's needed anymore?

Reply:
Honestly I don't. Unless there's a need to have a build step that doesn't pseudolocalize (as that's the only difference bewtween `build` and `build-prod`)

Reply:
Agree, you can probably go ahead and remove the prod one then.

Reply:
Removed build-prod script